miniapp/src/main/java/com/rakuten/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -0,0 +1,69 @@
+package com.rakuten.mobile.miniapp
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+import com.rakuten.tech.mobile.manifestconfig.annotations.ManifestConfig
+import com.rakuten.tech.mobile.manifestconfig.annotations.MetaData
+
+/**
+ * This initializes the SDK module automatically as the Content Providers are initialized
+ * at first before other initializations happen in the application's ecosystem.
+ */
+@Suppress("TodoComment", "UndocumentedPublicClass")
+class MiniappSdkInitializer : ContentProvider() {
+
+    @ManifestConfig
+    interface App {
+
+        /**
+         * Base Url for the mini app backend.
+         **/
+        @MetaData(key = "com.rakuten.mobile.miniapp.BaseUrl")
+        fun baseUrl(): String
+
+        /**
+         * Host app version for the mini app backend.
+         **/
+        @MetaData(key = "com.rakuten.mobile.miniapp.HostAppVersion")
+        fun hostAppVersion(): String
+
+        /**
+         * App Id assigned to host App.
+         **/
+        @MetaData(key = "com.rakuten.mobile.ras.AppId")
```

Comment:
The key for App ID has to be `com.rakuten.tech.mobile.ras.AppId` and the key for Subscription Key has to be `com.rakuten.tech.mobile.ras.ProjectSubscriptionKey` so that they are the same as our other SDKs.

Actually, the package name for this SDK should be `com.rakuten.tech.mobile` and not `com.rakuten.mobile` so maybe this is a good chance to change it. The package for Bintray is already `com.rakuten.tech.mobile.miniapp` so it should be okay to change.

- [ ] Change manifest key values to `com.rakuten.tech.mobile`
- [ ] Change package name to `com.rakuten.tech.mobile.miniapp` and sample app to ` `com.rakuten.tech.mobile.miniapp.testapp`